ozaria/site/components/interactive/PageInteractive/common/BaseButton.vue

```diff
@@ -0,0 +1,42 @@
+<script>
```

Comment:
Should this live in a common components dir?  Is this specific to interactives?

Reply:
This is currently specific to interactives.

Reply:
Can check with Shasha if this button is going to be similar for all systems? Else it seems specific for now and the name of class `ozaria-btn` can also be changed to be specific to interactives to avoid confusion.